src/en/madarascans/src/eu/kanade/tachiyomi/extension/en/madarascans/MadaraScans.kt

```diff
@@ -0,0 +1,26 @@
+package eu.kanade.tachiyomi.extension.en.madarascans
+
+import androidx.preference.PreferenceScreen
+import eu.kanade.tachiyomi.multisrc.mangathemesia.MangaThemesiaAlt
+import eu.kanade.tachiyomi.multisrc.mangathemesia.MangaThemesiaPaidChapterHelper
+
+class MadaraScans : MangaThemesiaAlt(
```

Comment:
```suggestion
class MadaraScans : MangaThemesia(
```
use base MangaThemesia as the site doesn't have random urls

Reply:
would need to also implement `ConfigurableSource` interface to get `setupPreferenceScreen`

Reply:
Sorry, I’m a beginner in Kotlin.
I removed the toggle and just hide paid chapters by default